messaging-experiments.json

```diff
@@ -412,5 +412,237 @@
         }
       ]
     }
+  },
+  {
+    "id": "PROTECTION-RETENTION-EXPERIMENT-RELEASE",
+    "enabled": true,
+    "filter_expression": "(env.channel == 'nightly' || env.channel == 'default')",
```

Comment:
For nightly only right now.

Reply:
Yep. Nit: the parentheses are redundant.